docs/cosmos-hub/ledger.md

```diff
@@ -1,76 +1,167 @@
+```bash
+$ gaiacli keys show <keyName> -d
+```
 
-## Ledger Support for Account Keys
+Confirm that the address displayed on the device matches that displayed when you added the key. To review the address again run `$ gaiacli keys list`.
 
-At the core of a Ledger device, there is a mnemonic that is used to generate private keys. When you initialize you Ledger, a mnemonic is generated. 
 
-::: danger
-**Do not lose or share your 24 words with anyone. To prevent theft or loss of funds, it is best to ensure that you keep multiple copies of your mnemonic, and store it in a safe, secure place and that only you know how to access. If someone is able to gain access to your mnemonic, they will be able to gain access to your private keys and control the accounts associated with them.**
-:::
+Note the gaiacli help commands are nested. So `$ gaiacli` will output docs for the top level commands (status, config, query, and tx. You can access documentation for sub commands with further help commands.
+`$ gaiacli query --help`
+`$ gaiacli tx --help`
 
-This mnemonic is compatible with Cosmos accounts. The tool used to generate addresses and transactions on the Cosmos Hub network is called `gaiacli`, which supports derivation of account keys from a Ledger seed. Note that the Ledger device acts as an enclave of the seed and private keys, and the process of signing transaction takes place within it. No private information ever leaves the Ledger device. 
 
-To use `gaiacli` with a Ledger device you will need the following:
+### Connect to a fullnode
+Next, you need to configure gaiacli with the URL of a Cosmos fullnode and the appropriate chain_id. In this example we connect to the public load balanced fullnode operated by Chorus One on the cosmoshub-2 chain. But you can point your `gaiacli` to any Cosmos fullnode. Be sure that the `chain_id` is set to the same chain as the fullnode. 
 
-- [A Ledger Nano with the `COSMOS` app installed and an account](./delegator-guide-cli.md#using-a-ledger-device)
-- [A running `gaiad` instance connected to the network you wish to use.](./delegator-guide-cli.md#accessing-the-cosmos-hub-network)
-- [A `gaiacli` instance configured to connect to your chosen `gaiad` instance.](./delegator-guide-cli.md#setting-up-gaiacli)
+`$ gaiacli config node https://cosmos.chorus.one:26657`   
+
+`$ gaiacli config chain_id cosmoshub-2`
+
+Test your connection with a query such as:
+
+`$ gaiacli query staking validators`
+
+*NOTE:* You can choose to run your own fullnode locally [read more here.](https://cosmos.network/docs/cosmos-hub/join-mainnet.html#setting-up-a-new-node).
+
+
+## Sign a transaction
+You are now ready to start signing and sending transactions. Send a transaction with gaiacli using the `tx send` command. 
+`$ gaiacli tx send --help` to see all available options.
+
+*NOTE:* Be sure to unlock your device with the PIN and open the Cosmos app before trying to run these commands
```

Comment:
this should be a `:: tip`. See https://vuepress.vuejs.org/guide/markdown.html#custom-containers